ui/app/templates/components/database-role-setting-form.hbs

```diff
@@ -8,10 +8,16 @@
   {{#if this.settingFields}}
     <div class="form-section">
       {{#each this.settingFields as |attr|}}
-        {{#if (and (eq @mode "edit") (eq attr.name "username"))}}
+        {{#if (and (eq @mode "edit") (or (eq attr.name "skip_import_rotation") (eq attr.name "username")))}}
```

Comment:
no need to change, just an fyi there's also an [includes](https://github.com/hashicorp/vault/blob/main/ui/app/templates/components/transit-key-action/datakey.hbs#L71) handlebar helper than can sometimes be helpful when we're looking for multiple params. 😄